Pod/Classes/TableViewHelpers.swift

```diff
@@ -30,23 +30,23 @@ extension UITableView {
 }
 
 public enum ReusableCell<Cell> {
-    case Class(identifier: String)
-    case Nib(identifier: String, nibName: String, bundle: NSBundle?)
+    case `class`(identifier: String)
```

Comment:
@codecaffeine thoughts on this? It shouldnt be `.class` but something is up. Is that apples new way of saying class